### ISSUE TITLE
Add @types/webxr to repo instead of duplicating & putting unknown

### ISSUE THREAD

**Issue by joshuaellis** [three-types/three-ts-types#31]
### Describe the feature you'd like:

A lot of the `XR` classes such as XREstimatedLight would benefit *hugely* from using `@types/webxr` instead of us putting `unknown` or some fudged class.

```ts
export class SessionLightProbe<TLight extends THREE.Group = THREE.Group> {
    xrLight: TLight;
    renderer: WebGLRenderer;
    lightProbe: unknown; // here would be helpful
    xrWebGLBinding: unknown | null;  // here would be helpful
    estimationStartCallback: () => void;
    frameCallback: (this: SessionLightProbe, time: number, xrFrame: XRFrame) => void;

    constructor(
        xrLight: TLight,
        renderer: WebGLRenderer,
        lightProbe: unknown,
        environmentEstimation: boolean,
        estimationStartCallback: () => void,
    );

    updateReflection: () => void;

    onXRFrame: (time: number, xrFrame: XRFrame) => void;

    dispose: () => void;
}
```

We could copy the types, but then those types link to other types in the library, why duplicate?

**Comment by CodyJasonBennett** [comment]
Related types are enumerated here: [@types/webxr/index.d.ts](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/webxr/index.d.ts).

**Comment by joshuaellis** [comment]
@CodyJasonBennett, are you suggesting we just copy them in?

**Comment by CodyJasonBennett** [comment]
That didn't come to mind, but I can't find anything there for this new AR lighting.

This was mostly copied from Mozilla docs, so I wonder if that's fine to do on our own.

**Comment by CodyJasonBennett** [comment]
I'm not seeing anything for the new AR lighting.

**Comment by Methuselah96** [comment]
Yeah, the types in `@types/webxr` aren't up-to-date, but we could help improve `@types/webxr` so that other people don't run into similar issues.

**Comment by joshuaellis** [comment]
@CodyJasonBennett, @Methuselah96 referenced this as a potential [source](https://immersive-web.github.io/lighting-estimation/#xrlightprobe-interface) in a pr, we could update the types for webxr and they'll trickle back in

**Comment by elalish** [comment]
If it's helpful, I have a set of webxr types here that might be a good place to start? https://github.com/google/model-viewer/blob/master/packages/model-viewer/src/types/webxr.ts

I'd rather get them from this repo than maintain my own set.

**Comment by CodyJasonBennett** [comment]
@elalish, thanks. That looks more correct, for sure.

Are we waiting for a release at `@types/webxr`, or should I go ahead with a PR there?

**Comment by joshuaellis** [comment]
I would make a PR there tbh.

**Comment by Methuselah96** [comment]
I think we should be making a PR on DefinitelyTyped. I can't tell if that's what was decided above or not.

**Comment by joshuaellis** [comment]
> I think we should be making a PR on DefinitelyTyped. I can't tell if that's what was decided above or not.

sorry, I wasn't clear, that's what I was suggesting 😅

**Comment by joshuaellis** [comment]
resolved by #218
